projects/ui-components/lf-repository-browser/lf-repository-browser.component.spec.ts

```diff
@@ -11,7 +11,8 @@ import { MatDialog } from '@angular/material/dialog';
 import { AppLocalizationService, LfLoaderComponent } from '@laserfiche/lf-ui-components/internal-shared';
 import { LfRepositoryBrowserComponent } from './lf-repository-browser.component';
 import { LfTreeNodeService, LfTreeNode } from './ILfTreeNodeService';
-import { LfSelectionListModule } from '../lf-selection-list/lf-selection-list.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ColumnDef, LfSelectionListComponent, LfSelectionListModule, RepositoryBrowserData } from '@laserfiche/lf-ui-components/lf-selection-list';
 
 const rootTreeNode: LfTreeNode = {
```

Comment:
add in test that if a column is not resizeable, should not be able to resize
and if a column is not sortable, should not be able to sort
if a column is sortable, should sort